lib/elastic_apm/subscriber.rb

```diff
@@ -54,14 +54,15 @@ def start(name, id, payload)
     end
     # rubocop:enable Metrics/MethodLength
 
-    def finish(_name, id, _payload)
+    def finish(name, id, payload)
       # debug "AS::Notification#finish:#{name}:#{id}"
       return unless (transaction = @agent.current_transaction)
 
       while (notification = transaction.notifications.pop)
         next unless notification.id == id
 
         if (span = notification.span)
+          span.original_backtrace ||= @normalizers.backtrace(name, payload)
```

Comment:
It's probably very little overhead, but as this is something that's potentially happening many times per second, we could skip the few allocations by checking `config.span_frames_min_duration?` before calling this?

Reply:
ok, done!